rllib/policy/torch_policy.py

```diff
@@ -598,9 +599,32 @@ def optimizer(
     @override(Policy)
     @DeveloperAPI
     def export_model(self, export_dir: str) -> None:
-        """TODO(sven): implement for torch.
+        """Exports the Policy's Model to local directory for serving.
+
+        Creates a TorchScript model and saves it.
+
+        Args:
+            export_dir (str): Local writable directory or filename.
         """
-        raise NotImplementedError
+        dummy_inputs = self._lazy_tensor_dict(self._dummy_batch.data)
+        # Provide dummy state inputs if not an RNN (torch cannot jit with
+        # returned empty internal states list).
+        if "state_in_0" not in dummy_inputs:
```

Comment:
Why does torch jit require state_in even if the model is not an RNN?

Reply:
Agree, this is a total hack. Torch requires the output to be some tensor (or nested struct of tensors), but NOT an empty list :/ That's why we need to fake it here. One more reason to keep **thinking** about a possible new ModelV3 API :)

Reply:
torch jit doesn't require state_in, it's the self.model that requires it.